### ISSUE TITLE
Bookmarks bar in compact mode is huge in FF 108

### ISSUE THREAD

**Issue by UncleScrooge85** [black7375/Firefox-UI-Fix#539]
Hello,
starting from FF 108, the bookmarks bar in compact mode is now as high as in standard mode, while tabs bar and address bar are still resizing according to the selected mode.

I'm using the photon-style fix.
Is it possible to update the fix so that the bookmarks bar will reduce its height again when using compact mode?
Thank you!

**Comment by black7375** [comment]
It looks normally in my environment.
It's patched in version [`v6.5.0`](https://github.com/black7375/Firefox-UI-Fix/releases/tag/v6.5.0), so try using the latest version.
![image](https://user-images.githubusercontent.com/25581533/208498512-2032b42f-2efc-48bf-85af-68ef21588077.png)

- https://github.com/black7375/Firefox-UI-Fix/blob/master/src/padding/_bookmarkbar.scss
- https://github.com/black7375/Firefox-UI-Fix/commit/b754c9ebc5245ed568d6d6adda1632a7af2740da

**Comment by UncleScrooge85** [comment]
Yes, the latest version improves the situation a little bit, but it's still higher than it was until FF 107.
This is what I see with the current version. The bookmarks bar has 2 lines of pixels over the text and 4 lines of pixels below the text. I'm pretty sure that the space below was lower before, either 2 or 3 lines of pixels but not 4 lines for sure.

![Cattura](https://user-images.githubusercontent.com/102796534/208630166-8cc9edfe-f165-4492-a306-f17975bf20ab.JPG)

**Comment by black7375** [comment]
Strangely, it's not reproduced
![image](https://user-images.githubusercontent.com/25581533/208762321-894b11fd-d019-4778-ac4d-01705be21d6c.png)

**Comment by Noceti** [comment]
Hi,
same thing after updating to newest Firefox.
Here's the differences between FF 108.0.1 vs 107.0.1:

![FF108vs107](https://user-images.githubusercontent.com/121315318/209398319-960f5e90-0428-418c-a62a-80a78fadd49d.jpg)

**Comment by black7375** [comment]
Try to latest commit

**Comment by Noceti** [comment]
> Try to latest commit

The difference is minimal.
Maybe the problem is with the gray overlay when you hower the cursor over the links:

![overlay](https://user-images.githubusercontent.com/121315318/209864065-b2c68a1d-1fea-4221-a95b-d7036f4da437.jpg)

**Comment by black7375** [comment]
@Noceti Please check one thing.

AppMenu - More Tools - Customize Toolbar
![image](https://user-images.githubusercontent.com/25581533/209947412-f2ec572e-2cdb-45a7-8265-e8b4df7503e4.png)

Is the compact mode right?
If that's right, it's `18px`. If we reduce further, it matches the size of the icon(`16px`), so there is a problem.
![image](https://user-images.githubusercontent.com/25581533/209947630-c9b77816-9a25-4d82-b50d-caf770f590a4.png)

**Comment by Noceti** [comment]
The compact mode is checked.

Here's the screenshot from the Customize Toolbar:
![comp](https://user-images.githubusercontent.com/121315318/209990089-3c6ea6c0-c02c-4937-8679-25404d40c1ab.jpg)
